### ISSUE TITLE
Production image for SLES 12 SP3 doesn't deploy properly

### ISSUE THREAD

**Issue by jeffaco** [SUSE-Enceladus/azure-li-services#129]
Production image `SLES12-SP3-SAP-Azure-LI-BYOS.x86_64-1.1.3-Build1.15.raw` doesn't seem to deploy properly. When deploying, every single service fails and I can't log in.

The console shows the following output:

<img width="973" alt="Screen Shot 2019-03-22 at 10 47 05 AM" src="https://user-images.githubusercontent.com/13700943/54842715-e48d2380-4c8f-11e9-813c-f3269fc149e3.png">

Attached is the [YAML](https://github.com/SUSE-Enceladus/azure-li-services/files/2997693/suse_firstboot_config.yaml.txt) and [workload.log](https://github.com/SUSE-Enceladus/azure-li-services/files/2997695/workload.log) files.

Note that the [YAML](https://github.com/SUSE-Enceladus/azure-li-services/files/2997693/suse_firstboot_config.yaml.txt) file worked fine with the SLES 12 SP4 (`SLES12-SP4-SAP-Azure-LI-BYOS.x86_64-1.0.2-Build1.15.raw`) and the SLES 15 (`SLES15-SAP-Azure-LI-BYOS.x86_64-1.0.3-Build1.12.raw`) images.

This is a bit of a showstopper. Advice?

**Comment by schaefi** [comment]
To my understanding the build in the SUSE namespace from where Robert took the image is broken

* SUSE:SLE-12-SP3:Update:PubClouds/SLES12-SP3-SAP-Azure-LI-BYOS

```
  <arch>x86_64</arch>
  <error>unresolvable: have choice for libpmemblk.so.1()(64bit) needed by fio: libpmemblk-devel libpmemblk1, have choice for libpmemblk.so.1(LIBPMEMBLK_1.0)(64bit) needed by fio: libpmemblk-devel libpmemblk1</error>
  <debuginfo>1</debuginfo>
```

I have no idea why this is the case in the SUSE namespace. We need to investigate this.

The reason that it does not build causes the build to be outdated:

* osc buildlog images x86_64 | grep azure-li-services

```
[  106s] azure-li-services-1.1.20-1.11.1.noarch (usr_src_packages_SOURCES_repos_SUSE:SLE-12:Update_standard): 
```

So the version of azure-li-services is old and does not understand the yaml as you used it.
This is the reason for the overall errors you saw

So in short the production build here is not working and should have never been released as such.

@rjschwei 
I really dislike that obs keeps old builds and does not wipe them if new build results are expected.
Is there a way to setup the project builds in the SUSE namepsace in a way that it either contains the latest build result or nothing ? If so that would prevent releasing old stuff

**Comment by schaefi** [comment]
As I don't have permissions in the SUSE namespace and the error is not related to my package, I can't fix this.

**Comment by schaefi** [comment]
Robert deleted the wrong image from the Azure container. The build issue has been reported. We are waiting for the build team to address our request.

I keep this open as a reference.

@rjschwei please close as soon as you've uploaded a new image

Thanks

**Comment by schaefi** [comment]
Thanks Robert for providing a new production image.

I've uploaded the image to Azure and sent @jeffaco an e-mail with further details
